content/v2/openapi.yml

```diff
@@ -2776,6 +2781,27 @@ components:
       description: Only include results with a type field exactly matching the given string
       schema:
         type: string
+    FilterRegistrantChangeState:
+      name: state
+      in: query
+      required: false
+      description: Only include results with a state field exactly matching the given string
+      schema:
+        type: string
+    FilterDomainId:
+      name: domain
+      in: query
+      required: false
+      description: Only include results with a domain_id field exactly matching the given string
+      schema:
+        type: string
+    FilterContactId:
+      name: contact
+      in: query
+      required: false
+      description: Only include results with a contact_id field exactly matching the given string
```

Comment:
Same as the above. `contact_id` vs `contact`

Reply:
Fixed :+1: